code/_globalvars/lists/flavor_misc.dm

```diff
@@ -16,6 +16,8 @@ GLOBAL_LIST_EMPTY(undershirt_m)	 //stores only undershirt name
 GLOBAL_LIST_EMPTY(undershirt_f)	 //stores only undershirt name
 	//Socks
 GLOBAL_LIST_EMPTY(socks_list)		//stores /datum/sprite_accessory/socks indexed by name
+	//Body Sizes
+GLOBAL_LIST_INIT(body_sizes, list("Normal", "Short", "Tall"))
```

Comment:
this global list isn't needed since it's only used once. You can instead make it a static list.

Reply:
Consistency with other feature-preferences. its 3 strings. This is a tiny PR, and preferences already need a major refactor. I get the "dont add to the problem" but come on.

Reply:
Remove the list, you are converting this to numbers later and it's saved by a var

Reply:
It's for parity with all other preferences. It's used in preference code.